standards/application/privatev1.md

```diff
@@ -0,0 +1,417 @@
+
+Protobuf does not guarantee byte-identical outputs for multiple serializations of the same logical message.
+Because of this, the `frame_id` represents the hash of specific encoded bytes rather than an abstract frame structure.
+Implementations MUST compute `frame_id` from the actual bytes being transmitted to ensure sender and receiver derive identical identifiers.
+
+
+## Protocol Operation
+
+PRIVATE1 processes messages through a three-stage pipeline where each stage's output becomes the next stage's input.
+The specific ordering of these stages is critical for maintaining security properties while enabling efficient operation.
+
+```mermaid
+flowchart TD
+    C("Content"):::plain
+    S(Segmentation)
+    R(Reliability)
+    E(Encryption) 
+     D(Delivery):::plain
+    C --> S --> R --> E --> D
+
+    classDef plain fill:none,stroke:transparent;
+```
+**Pipeline Stages:**
+- **Segmentation**: Divides content into transport-appropriate fragments
+- **Reliability (SDS)**: Adds tracking metadata for delivery detection and ordering
+- **Encryption (Double Ratchet)**: Provides confidentiality, authentication, and forward secrecy
+
+
+
+### Segmentation
+
+While PRIVATE1 itself has no inherent message size limitation, practical transport mechanisms typically impose maximum payload sizes.
+
+**Why Segment Before Encryption**
+
+Segmenting after encryption would force the transport layer to handle fragmentation of ciphertext blobs, creating several problems.
+- Transport-layer segmentation would require buffering all segments before any can be authenticated, increasing the DOS attack surface.
+- Unauthenticated segment reassembly opens the door to malicious segment injection and substitution attacks.
+- Unencrypted segmentation metadata reveals size and other metadata about the content in transit. 
+
+**Why Segment Before Reliability**
+
+Placing segmentation after reliability tracking would mean retransmission of a dropped segment would require re-broadcasting the entire frame.
+By segmenting first, the reliability layer can track individual segments and request retransmission of only the missing fragments.
+
+**Implementation**
+
+The segmentation strategy used is defined by [!TODO: Flatten link once completed](https://github.com/waku-org/specs/pull/91)
+
+Implementation specifics:
+- Error correction is not used, as reliable delivery is already provided by lower layers. 
+- `segmentSize` = `max_seg_size`
+- All payloads regardless of size are wrapped in a segmentation message.
+ 
+### Message Reliability
+
+Scalable Data Sync (SDS) is used to detect missing messages, provide delivery confirmation, and handle retransmission of payloads.
+SDS is implemented according to the [specification](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/sds.md).
+
+**SDS Field Mappings**
+
+The following mappings connect PRIVATE1 concepts to SDS fields:
+
+- `sender_id`: !TODO: This requires PRIVATE1 to be identity aware
```

Comment:
Unsure exactly what the best course of action is here. Recent changes to SDS requires an sender_id be attached the the SdsMessage. 

This then brings in a concept of identity to this specification. Its definitely possible to introduce Identity concepts in order to pass them through to SDS, however it seems like a lot of added complexity